es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = `${number}  ${street}`;
```

Comment:
The number of spaces here changed in the refactor, please fix it:

---

![image](https://user-images.githubusercontent.com/16143771/156278595-f0e21c85-600f-49cb-acc5-98734467548f.png)

---

![image](https://user-images.githubusercontent.com/16143771/156278646-05ffe8a9-4e68-42ee-b531-ec5703cb9a67.png)